### DISCUSSION TITLE
scrolling indicator

### DISCUSSION THREAD

**Discussion by tizu69** [charmbracelet/huh#591] [Ideas]
**Is your feature request related to a problem? Please describe.**
scrolling in multiselect lists requires me to change the subtitle to like "hey u can scrolL" because otherwise the user wouldn't know

**Describe the solution you'd like**
probably a
```
┃ Select scripts to include in your package                                                                        
┃ You may scroll to discover more scripts! (this is what I did as a temp solution)
┃ > [ ] server_scripts/fffff copy 1.js                                                                             
┃   [ ] server_scripts/fffff copy 10.js                                                                            
┃   [ ] server_scripts/fffff copy 11.js                                                                            
┃   [ ] server_scripts/fffff copy 12.js                                                                            
┃   [ ] server_scripts/fffff copy 13.js                                                                            
┃   [ ] server_scripts/fffff copy 14.js                                                                            
┃   [ ] server_scripts/fffff copy 15.js                                                                            
V   [ ] server_scripts/fffff copy 16.js   
```
or just have the cursor not reach the end unless its the last item (so like, scroll with a 1-item gap)

**Describe alternatives you've considered**
text

**Additional context**
i like apples

**Comment by bashbunni** [comment]
Hey @tizu69, thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

I really like this idea! Thanks for bringing it up :)
